app/routes/wow.item-data.$itemId.tsx

```
@@ -99,172 +168,191 @@
+            <div className="flex flex-col max-w-full">
+              {listing.salesPerDay !== -1 && (
+                <Differences
+                  diffTitle="Sales Per Day"
+                  diffAmount={listing.salesPerDay.toLocaleString()}
+                  className="bg-blue-100 text-blue-900 font-semibold dark:bg-blue-600 dark:text-gray-100"
+                />
+              )}
+              {listing.percentChange !== -1 && (
+                <Differences
+                  diffTitle="Percent Change"
+                  diffAmount={listing.percentChange.toLocaleString() + '%'}
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)